step-controller/step-controller-server/src/main/java/step/controller/services/entities/AbstractEntityServices.java

```diff
@@ -57,6 +57,16 @@ public void init() throws Exception {
     public T get(@PathParam("id") String id) {
         return accessor.get(id);
     }
+    
+    @Operation(operationId = "find{Entity}sByIds", description = "Returns the list of entities matching the provided attributes")
```

Comment:
Change description "Returns the list of entities matching the provided attributes" to "Returns the list of entities for the provided list of IDs"